src/components/PanelMenuWrapper.js

```diff
@@ -10,8 +10,8 @@ class PanelsWithSidebar extends Component {
     const opts = this.computeMenuOptions(props);
 
     this.state = {
-      group: opts[0].name,
-      panel: opts[0].panels[0],
+      group: opts[1].name,
```

Comment:
this is a bit rough... :) maybe we can iterate over the children or something and grab the first one?